docs/UserGuide.md

```diff
@@ -255,7 +395,7 @@ Your tasks are automatically saved in *data.csv* which is located next to the pr
 | Add task          | `add n/NAME [t/TIME] [d/DURATION] [D/DEADLINE] [r/RECURRENCE] [i/IMPORTANCE] [a/ADDITIONAL-NOTES]`            |
 | List tasks        | `list [f/FORECAST] [i/IMPORTANCE]`                                                                          |
 | Mark task as done | `done INDEX`                                                                                                |
-| Edit task         | `edit INDEX [n/NAME] [t/TIME] [d/DURATION] [D/DEADLINE] [r/RECURRENCE] [i/IMPORTANCE] [a/ADDITIONAL-NOTES]` |
+| Edit task         | `edit TASKID [n/NAME] [t/TIME] [d/DURATION] [D/DEADLINE] [r/RECURRENCE] [i/IMPORTANCE] [a/ADDITIONAL-NOTES]` |
```

Comment:
Will the reader read "TASKID" as one word "taskid" :thinking:? Maybe "TASK_ID" or "TASK-ID" would be clearer.